### ISSUE TITLE
Accessibility Issue: “% Complete” column is graphical, and not readable for a screen reader.

### ISSUE THREAD

**Issue by taysea** [grommet/hpe-design-system#1584]
“% Complete” column is graphical, and not readable for a screen reader. 
This % Complete column is using color alone to convey information. If the color had a visual representation as well as color it would pass. Also, think about adding a numeric indication that could also be picked up by the screen reader.

<img width="771" alt="Screen Shot 2021-04-26 at 11 49 23 AM" src="https://user-images.githubusercontent.com/12522275/116135068-7d6ba580-a685-11eb-95ec-dbc7ad1b0624.png">
